Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -611,6 +611,7 @@
       - ClothingNeckMantleHOS
       - ClothingNeckMantleHOSShoulder # Corvax-Resprite: Old mantle
       - ClothingNeckMantleRD
+      - ClothingNeckMantleQM # Corvax Mantle
```

Под Resprite попадают как изменения спрайтов так и новые спрайты
И формат комментария должен быть такой:
`# Corvax-Тема: Опциональное описание`
```suggestion
      - ClothingNeckMantleQM # Corvax-Resprite
```